### ISSUE TITLE
when adding executables the checkbox for "force Load Libraries" will not uncheck

### ISSUE THREAD

**Issue by indeliblemersion** [ModOrganizer2/modorganizer#1768]
## The problem:
Briefly describe the issue you are experiencing. Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our [Discord server](https://discord.gg/6GKR9jZ).

## Environment:

* Mod Organizer Version that exhibits the issue:
* Last Mod Organizer Version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer:

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your latest USVFS log, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by UpstairsEmpire** [comment]
Can confirmt this problem is still occurring. 
MO 2.5.2 
Windows 11

**Comment by UpstairsEmpire** [comment]
https://gist.github.com/UpstairsEmpire/6b0f0965214a6f010925bae13a619f6f

**Comment by jimmon89** [comment]
I found a workaround!
1) Open the Executable Settings Window, it will have the title `Modify Executables`.
2) Go to the executable you desire to disable force loading of libraries.
3) Make sure that `Force load libraries (*)` is in fact enabled (yes, enabled, it'll make sense, i promise).
4) Click the button labeled `Configure Libraries` to the right of the checkbox and text for `Force load libraries (*)`.
5) A new window should open with the title `Force Load Settings`, on that window click `Add Row`.
6) On the newly created row from left to right there should be an unchecked checkbox, a text box with the default text of `Process name`, then a button with 3 dots, the another text box with the default text of `Library to load`, and finally another button with 3 dots, if there is, don't do anything other than click `Ok` to save and that will close that window.
7) Now back in the window with the title `Modify Executables`, uncheck the box to the left of the label `Force load libraries (*)`.
8) Finally, Click `Apply`.
9) The  `Force load libraries (*)` checkbox will now stay unchecked.

In short, the actual problem is caused by [this](https://github.com/ModOrganizer2/modorganizer/blob/925bade3157900303a45fff3b64e64ed9ee5b0d8/src/profile.cpp#L1110) line of code in `src/profile.cpp`.

It sets the checkbox for `Force load libraries (*)` to be enabled by default, but no configuration entry is created in settings.ini in your mo2 profile unless you add what is effectively an empty row in the `Force Load Settings` window.

Once that configuration entry is created, the setting will now save.

However, if you delete the empty row in the `Force Load Settings` window, some cleanup code elsewhere will remove the whole block in the `settings.ini` file and you're back to `Force load libraries (*)` being force enabled.
